README.md

```diff
@@ -1,47 +1,38 @@
+[![Build Status](https://travis-ci.org/ember-learn/guides-source.svg?branch=master)](https://travis-ci.org/ember-learn/guides-source)
+[![This project is using Percy.io for visual regression testing.](https://percy.io/static/images/percy-badge.svg)](https://percy.io/Ember/guides-app)
 
-## Ember Guides Source [![Build Status](https://travis-ci.org/ember-learn/guides-source.svg?branch=master)](https://travis-ci.org/ember-learn/guides-source)
+# Ember Guides Source
 
-This repository contains the written content
-for the [Ember.js Guides](https://guides.emberjs.com).
-Here, contributors can file issues and submit PRs to
-help improve the learning experience of other developers.
+This repository contains the written content for the [Ember.js Guides](https://guides.emberjs.com). Contributors can file issues and submit pull requests (PRs) to help improve everyone's learning experience.
+
+Looking for repositories for the other parts of [emberjs.com](https://emberjs.com)? Check out the [website](https://github.com/ember-learn/ember-website), [ember-api-docs](https://github.com/ember-learn/ember-api-docs), [super-rentals tutorial](https://github.com/ember-learn/super-rentals), [statusboard](https://github.com/ember-learn/statusboard), [deprecation-app](https://github.com/ember-learn/deprecation-app), and [styleguide](https://github.com/ember-learn/ember-styleguide).
 
-Looking for repositories for the other parts of [emberjs.com](https://emberjs.com)?
-Check out
-[website](https://github.com/ember-learn/ember-website),
```

Comment:
What is the rationale behind the spacing changes here?

Reply:
Hmm, good question. In general, in this PR, I preferred to not break a sentence midway (I assumed, by number of characters).

For the list of links, I did think not breaking the sentence can make it harder to separate the links from the rest of the sentence in Markdown.

I'm not sure which of the following would be preferred by many:

- Break out the links from the sentence (current)
- Don't break the sentence (proposed change)
- Make an actual list

Reply:
I don't really have a preference. I would be curious what others things is best.

Reply:
Yeah. Let's revert the style to what it was before (a new line per link). If someone doesn't like the style, we can ask them to open an issue for further discussion.